_src/pattern-library/sass/components/_buttons.scss

```diff
@@ -57,6 +57,10 @@
 .btn-base {
     padding: spacing-vertical(x-small) spacing-horizontal(base);
     font-size: font-size(base);
+
+    svg {
```

Comment:
Hmm, good to know that we need to also size the ``svg`` element here. I'd prefer us to use the ``.icon`` class rather than the HTML element in this selector. Please sync up in all of these areas.